src/api/tenderly/tenderlyApi.ts

```diff
@@ -186,3 +191,11 @@ function _contractName(name: string): string {
 
   return name
 }
+
+export function getAliasFromAddress(address: string, isUnknown = false): string {
+  const lowerCaseAddress = address.toLowerCase()
+
+  if (SPECIAL_ADDRESSES[lowerCaseAddress]) return SPECIAL_ADDRESSES[lowerCaseAddress]
+
+  return isUnknown ? abbreviateString(address, 6, 4) : ALIAS_TRADER_NAME
+}
```

Comment:
i don\t think this is related to tenderly, consider moving to a more central point

i.e. We might want to use it for resolving it also in the lists